test/plugins/test_player.py

```diff
@@ -549,7 +549,7 @@ def test_cmd_noidle(self):
             # Manually send a command without reading a response.
             request = client.serialise_command("idle")
             client.sock.sendall(request)
-            time.sleep(0.01)
+            asyncio.sleep(0.01)
```

Comment:
need to await the asynchronous sleep, for that need to make the test method async and for that need to make the class inherit from `unittest.IsolatedAsyncioTestCase` and after that every method will have to be made async. Highly suggest to make this a different pr to make test cases asynchronous.

Reply:
Ah, I missed that, might be why the test is failing. If this is a problem, I suggest waiting until I attempt to refactor tests to use pytest.

Reply:
In that case, I'll revert it back to time.sleep. And maybe leave a TODO mark here?

Reply:
Yeah, for `time.sleep` I believe it's the only one.

Reply:
Leave it as time.sleep then. And yeah we use it to run the tests but most of the test framework is still unittest. So it'd be good to use the features of the better framework.